tests/Kernel/fixtures/rendering.yml

```diff
@@ -308,6 +308,84 @@
       'iframe.ecl-file__video[src="https://ec.europa.eu/avservices/play.cfm?ref=I101631"]': 1
     equals:
       'div.ecl-file__caption': "Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit."
+- array:
+    '#type': pattern
+    '#id': gallery
+    '#fields':
+      items:
+        - image:
+            src: "http://img.jpg"
+            name: "img.jpg"
+            alt: "Example alt"
+            responsive: true
+          caption: "Example caption"
+          icon: 'camera'
+  assertions:
+    count:
+      '.ecl-gallery': 1
+      '.ecl-gallery .ecl-row': 1
+      '.ecl-gallery .ecl-row .ecl-gallery__item': 1
+      '.ecl-gallery .ecl-row .ecl-gallery__item .ecl-gallery__item-container': 1
+      '.ecl-gallery__item-container .ecl-gallery__image[src="http://img.jpg"]': 1
+      '.ecl-gallery__item-container .ecl-gallery__image[alt="Example alt"]': 1
+      '.ecl-gallery__item-container span.ecl-gallery__caption': 1
+      '.ecl-gallery__item-container span.ecl-icon': 1
+      '.ecl-gallery__item-container span.ecl-icon.ecl-icon--camera': 1
+    equals:
+      'span.ecl-gallery__caption': "Example caption"
+- array:
+    '#type': pattern
+    '#id': gallery
+    '#fields':
+      items:
+        - image:
+            src: "http://img_1.jpg"
+            name: "img_1.jpg"
+          icon: "camera"
+        - image:
+            src: "http://img_2.jpg"
+            name: "img_2.jpg"
+            responsive: false
+          icon: "audio"
+  assertions:
+    count:
```

Comment:
Shouldn't we use expression language here ? Just like in epoetry ?

Reply:
I will create the following ticket to use expression language for all testes on oe_theme, to be consistent.
Nice improvement.

Reply:
@drupol https://github.com/openeuropa/oe_theme/issues/180